spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/contentMetadata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
```

Comment:
why do we need these files?

Reply:
Not strictly necessary. But the idea was to have a complete moab so as not to have to understand how the moab versioning gem works (to know which files are and are not necessary) and to be the same as pres cat fixtures.

Reply:
Is that documented somewhere?

Reply:
It's a test fixture. Where / how would you suggest documenting?

Reply:
README added.